coprocessor/fhevm-engine/host-listener/src/cmd/mod.rs

```diff
@@ -735,7 +769,7 @@ impl InfiniteLogIter {
             } else {
                 return false;
             };
-        current_block_number > end_at_block
+        current_block_number > end_at_block as u64
```

Comment:
This won't work with end_of_block = -1 = 2*64 - 1

Reply:
good catch